### PR TITLE
fix:Made UI issues on stock module

### PR DESCRIPTION
**Pull request by FelixKiprotich350**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

**Issue 1: Date picker responsiveness**
Made the date picker in stock operations base details component responsive. The date picker for example when disposing does not keep the value selected from the date input. Its stack on the default value which is (today).
I have made that responsive

**Issue 2: Removed the repeating word stock** 
I have removed the repetitive stock word on the stock management module menu.

**Issue 3: Replacement of non drugs with pharmaceuticals**
The sections rendering the words non-drugs and drugs have ben replaced by pharmaceuticals and non pharmaceuticals respectively through translation

## Screenshots

**_For Date picker_**
[screen-capture.webm](https://github.com/openmrs/openmrs-esm-stock-management/assets/72152368/55cbca6c-a10c-4a51-a166-eb285df43a6e)

**_For repeating stock word in menu_**
image

**_For Translation of non drugs words_**
image

**_For expiring items overview page_**
image

### CODE REVIEW

**Comment by jabahum** [comment]
@FelixKiprotich350 lets fix the build errors

**Comment by donaldkibet** [comment]
@FelixKiprotich350 please resolve the conflict, we move this forward

**Comment by FelixKiprotich350** [comment]
> @FelixKiprotich350 please resolve the conflict, we move this forward
i have made changes on that you can check it out

**Comment by ojwanganto** [comment]
Please resolve the conflicts as well. You may need to rebase with master

File: src/stock-items/components/filter-stock-items/filter-stock-items.component.tsx [hunk 1]
```diff
@@ -19,8 +19,8 @@ const FilterStockItems: React.FC<FilterStockItemsProps> = ({
       className={styles.spacing}
     >
       <RadioButton labelText="All" value="" id="is-drug-all" />
-      <RadioButton labelText="Drugs" value="true" id="is-drug-drug" />
-      <RadioButton labelText="Other" value="false" id="is-drug-other" />
+      <RadioButton labelText="Pharmaceuticals" value="true" id="is-drug-drug" />
+      <RadioButton labelText="Non-Pharms" value="false" id="is-drug-other" />
```
Comment (makombe) [inline src/stock-items/components/filter-stock-items/filter-stock-items.component.tsx:23]:
```suggestion
      <RadioButton labelText="Non Pharmaceuticals" value="false" id="is-drug-other" />
```

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**